Author: Michele Baldessari
Description: comment out mod_log_config (built in) and rename mod_nss

dirsrv httpd.conf path fixes
--- a/admserv/cfgstuff/httpd-2.2.conf.in
+++ b/admserv/cfgstuff/httpd-2.2.conf.in
@@ -120,7 +120,7 @@ ThreadsPerChild 64
 LoadModule authz_host_module @moddir@/mod_authz_host.so
 LoadModule auth_basic_module @moddir@/mod_auth_basic.so
 LoadModule authn_file_module @moddir@/mod_authn_file.so
-LoadModule log_config_module @moddir@/mod_log_config.so
+#LoadModule log_config_module @moddir@/mod_log_config.so
 LoadModule env_module @moddir@/mod_env.so
 LoadModule mime_magic_module @moddir@/mod_mime_magic.so
 LoadModule unique_id_module @moddir@/mod_unique_id.so
@@ -132,7 +132,7 @@ LoadModule alias_module @moddir@/mod_ali
 LoadModule rewrite_module @moddir@/mod_rewrite.so
 LoadModule cgi_module @moddir@/mod_cgi.so
 LoadModule restartd_module     @admmoddir@/mod_restartd.so
-LoadModule nss_module         @nssmoddir@/libmodnss.so
+LoadModule nss_module         @nssmoddir@/mod_nss.so
 LoadModule admserv_module     @admmoddir@/mod_admserv.so
 
 ### Section 2: 'Main' server configuration
--- a/admserv/cfgstuff/httpd.conf.in
+++ b/admserv/cfgstuff/httpd.conf.in
@@ -119,7 +119,7 @@ ThreadsPerChild 64
 # LoadModule foo_module modules/mod_foo.so
 LoadModule access_module @moddir@/mod_access.so
 LoadModule auth_module @moddir@/mod_auth.so
-LoadModule log_config_module @moddir@/mod_log_config.so
+#LoadModule log_config_module @moddir@/mod_log_config.so
 LoadModule env_module @moddir@/mod_env.so
 LoadModule mime_magic_module @moddir@/mod_mime_magic.so
 LoadModule expires_module @moddir@/mod_expires.so
@@ -142,7 +142,7 @@ LoadModule file_cache_module @moddir@/mo
 LoadModule mem_cache_module @moddir@/mod_mem_cache.so
 LoadModule cgi_module @moddir@/mod_cgi.so
 LoadModule restartd_module     @admmoddir@/mod_restartd.so
-LoadModule nss_module         @nssmoddir@/libmodnss.so
+LoadModule nss_module         @nssmoddir@/mod_nss.so
 LoadModule admserv_module     @admmoddir@/mod_admserv.so
 
 ### Section 2: 'Main' server configuration
--- a/admserv/cfgstuff/httpd-2.4.conf.in
+++ b/admserv/cfgstuff/httpd-2.4.conf.in
@@ -120,7 +120,7 @@ ThreadsPerChild 64
 LoadModule authz_host_module @moddir@/mod_authz_host.so
 LoadModule auth_basic_module @moddir@/mod_auth_basic.so
 LoadModule authn_file_module @moddir@/mod_authn_file.so
-LoadModule log_config_module @moddir@/mod_log_config.so
+#LoadModule log_config_module @moddir@/mod_log_config.so
 LoadModule env_module @moddir@/mod_env.so
 LoadModule mime_magic_module @moddir@/mod_mime_magic.so
 LoadModule unique_id_module @moddir@/mod_unique_id.so
@@ -132,14 +132,14 @@ LoadModule alias_module @moddir@/mod_ali
 LoadModule rewrite_module @moddir@/mod_rewrite.so
 LoadModule cgi_module @moddir@/mod_cgi.so
 LoadModule restartd_module     @admmoddir@/mod_restartd.so
-LoadModule nss_module         @nssmoddir@/libmodnss.so
+LoadModule nss_module         @nssmoddir@/mod_nss.so
 LoadModule admserv_module     @admmoddir@/mod_admserv.so
 LoadModule mpm_worker_module    @moddir@/mod_mpm_worker.so
 LoadModule access_compat_module @moddir@/mod_access_compat.so 
 LoadModule authn_core_module    @moddir@/mod_authn_core.so 
 LoadModule authz_core_module    @moddir@/mod_authz_core.so 
 LoadModule authz_user_module    @moddir@/mod_authz_user.so 
-LoadModule unixd_module         @moddir@/mod_unixd.so
+#LoadModule unixd_module         @moddir@/mod_unixd.so
 
 ### Section 2: 'Main' server configuration
 #
